posts/posts-router.js

```diff
@@ -0,0 +1,97 @@
+const express = require('express');
+
+const Db = require('../data/db.js');
+
+const router = express.Router();
+
+// /api/posts
+router.post('/posts', (req, res) => {
+    const post = req.body
+    if (!post.name || !post.bio) {
+        res.status(400).json({ errorMessage: "Post not founds" })
+    } else {
+        Db.insert(user)
+            .then(posts => {
+                res.status(201).json(posts)
+            })
+            .catch(err => {
+                res.status(500).json({ error: "Posts Not Found" })
+            })
+    }
+})
+
+router.get('/', async(req, res) => {
+    try {
+        const db = await Db.find()
+        res.status(200).json(db)
+    } catch (err) {
+        console.log(err);
+        res.status(500).json({
+            message: 'Error retrieving the posts',
+        });
+    }
+})
+
+
+
+
+// gets a specific psot by specifying /api/posts/(insert id)
```

Comment:
good job with the comments, really simple but helps express a lot about your api. It's great way to communicate your intentions to another team member or yourself later on.